pages/api/reviews/model/ReviewSchema.ts

```diff
@@ -0,0 +1,22 @@
+import { model, models, Schema, Types } from 'mongoose';
+
+interface IReview {
+    revieweeId: Types.ObjectId;
+    reviewerId: Types.ObjectId;
```

Comment:
@fazlerabby07 userId ta k foreign key rakhle bhalo hoy. ORelse it would be hard to find k kake review dilo.

Reply:
@sifatul fix it.